app/src/screens/About/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Linking,
+  TouchableHighlight,
+} from 'react-native';
+import { getEnvVariable } from '../../util/env';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#F4F4F6',
+    height: '100%',
+    display: 'flex',
+    flex: 1,
+    alignContent: 'center',
+    marginTop: 100,
+    marginHorizontal: 40,
+  },
+  header: {
+    color: '#121C2D',
+    fontSize: 24,
+    fontWeight: '700',
+    marginBottom: 10,
+  },
+  body: { flex: 1 },
+  section: {
+    marginBottom: 50,
+  },
+  highlight: {
+    color: '#0263E0',
+  },
+  channel: {
+    color: '#0263E0',
+    textDecorationLine: 'underline',
+  },
+  issueDescription: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+  label: {
+    fontWeight: '700',
+  },
+  footer: {
+    height: 50,
+  },
+});
+
+const About: React.FC = () => {
+  const slackURL = getEnvVariable('SLACK_URL');
```

Comment:
These never change, so let's move them out of the component body. If we leave it here, it will get called repeatedly. If we don't want to move it out of this functional component, we can use a memoization hook, but that still means it gets called for every component mount at the very least.

Reply:
I get the following error when moving out of component body, any ideas:
<img width="899" alt="Screen Shot 2023-07-10 at 10 53 21 AM" src="https://github.com/twilio/twilio-voice-react-native-app/assets/35968892/f6f7ee48-2895-4b76-b45d-9aed6f417e28">

Added `usememo` hook

Reply:
Hmm, not sure. I think the `useMemo` hook is fine though if it solves it. Unless you're saying using the `useMemo` hook causes this?